src/pages/Payslip.jsx

```diff
@@ -100,9 +119,24 @@ function Payslip() {
     const onPayCodeChange = event => dispatch({ type: ACTIONS.PAYCODE_CHANGED, payload: event.target.value })
     const onCurrencyCode = event => dispatch({ type: ACTIONS.CURRENCY_CHANGED, payload: event.target.value })
     const onTotalAmountChange = event => dispatch({ type: ACTIONS.TOTAL_AMOUNT, payload: event.target.value })
-    const onAccountReceivableChange = event =>
-        dispatch({ type: ACTIONS.ACCOUNT_RECEIVABLE, payload: event.target.value })
-
+    const appendZeros = value => {
```

Comment:
@MrGeminus 

I think this could be moved to component as in this app we will have it only with these cases. 

What do you think ?

Reply:
If we won't reuse it elsewhere, moving it to the component makes sense. I agree.